rules/Php80/Rector/Class_/AnnotationToAttributeRector.php

```diff
@@ -118,6 +119,10 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
+        if ($this->annotationsToAttributes === []) {
+            throw new ConfigurationException(sprintf('The "%s" rule requires configuration.', self::class));
+        }
+
```

Comment:
It seems people add this rule and expect to convert all annotations to attributes. Then they come to Github issue to report.

Instead, we can make this feedback loop shorter and let them now immediately :)